app/frontend/js/map.js

```diff
@@ -98,9 +109,20 @@ export function displayMap (sbi, parcels, coordinates) {
   map.addInteraction(selectPointerMove)
 
   selectClick.on('select', function (e) {
-    window.location.href = `/parcel?sbi=${sbi}&sheetId=${e.selected[0].values_.sheet_id}&parcelId=${e.selected[0].values_.parcel_id}`
+    window.location.href = `/parcel?sbi=${sbi}&sheetId=${e.selected[0].values_.sheet_id}&parcelId=${e.selected[0].values_.parcel_id}&mapStyle=${select.value}`
   })
 
+  var select = document.getElementById('layer-select')
+
+  function onChange () {
+    var style = select.value
+    for (var i = 0, ii = layers.length - 1; i < ii; ++i) {
+      layers[i].setVisible(mapStyles[i] === style)
+    }
+  }
```

Comment:
Suggest changing this to:

```
const style = select.value
  const totalLayers = layers.length - 1
  for (let i = 0; i < totalLayers; ++i) {
    layers[i].setVisible(mapStyles[i] === style)
  }
```
It makes it more readable and SonarCloud/StandardJS will have issue with declaring values on one line as it can have side effects.

ie instead of:
```
let i, ii
```
you would have:
```
let i
let ii
```